.travis.yml

```
@@ -111,6 +111,13 @@ jobs:
       script: scripts/deploy.js omail-link-processor
       install: skip
       env: [ENVIRONMENT=production RANCHER_CLUSTERID=c-gxsr7]
+
+    - stage: production
+      name: Algolia Sync
+      script: scripts/deploy.js GRAPHQL_URI
```

```suggestion
      script: scripts/deploy.js algolia-sync
```
Pretty sure this is wrong :P

---

haha fixed now... nothing to see here :P